docs/topics/gradle/gradle-configure-project.md

```diff
@@ -386,6 +386,63 @@ integrationTestCompilation {
 Here, the `integrationTest` compilation is associated with the `main` compilation that gives access to `internal`
 objects from functional tests.
 
+### Configure with Java JPMS enabled
+
+To make the Kotlin Gradle plugin work with [JPMS](https://www.oracle.com/corporate/features/understanding-java-9-modules.html), 
+add the following lines to your build script and replace `YOUR_MODULE_NAME` with a reference to your module, for example, 
+`org.company.module`:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+        
+```kotlin
+// Add the following three lines if you use a Gradle version less than 7.0
+java {
+    modularity.inferModulePath.set(true)
+}
+
+tasks {
+    named("compileJava", JavaCompile::class.java) {
```

Comment:
Nit: Better to use here also `tasks.named` to be almost similar to Groovy example